src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -79,6 +79,21 @@ class RoutedSavableReviewPage extends React.Component {
       formConfig?.customText?.finishAppLaterMessage ||
       FINISH_APP_LATER_DEFAULT_MESSAGE;
     const downtimeDependencies = get('downtime.dependencies', formConfig) || [];
+    const saveLink = (
+      <SaveFormLink
+        locationPathname={location.pathname}
+        form={form}
+        formConfig={formConfig}
+        user={user}
+        pageList={pageList}
```

Comment:
Actually, I'm not seeing `pageList` used in `SaveFormLink`, so do we even need to pass this?

https://github.com/department-of-veterans-affairs/vets-website/blob/ce71f36d84d920a8321f1306fb507b5692902157/src/platform/forms/save-in-progress/SaveFormLink.jsx#L23

Reply:
Ah, it looks like it gets passed in the spread:

https://github.com/department-of-veterans-affairs/vets-website/blob/ce71f36d84d920a8321f1306fb507b5692902157/src/platform/forms/save-in-progress/SaveFormLink.jsx#L90

Since this is purely a dev accomodation, I'm curious if we could accomplish this another way so that we can not have to pass `saveLink` down a bunch.